backend/packages/Upgrade/src/api/services/FeatureFlagService.ts

```diff
@@ -110,36 +140,119 @@ export class FeatureFlagService {
         throw error;
       }
 
-      // creating variations docs
-      const variationDocsToSave =
-        variations &&
-        variations.length > 0 &&
-        variations.map((variation: FlagVariation) => {
-          variation.id = variation.id || uuid();
-          variation.featureFlag = featureFlagDoc;
-          return variation;
-        });
-
-      // saving variations
-      let variationDocs: FlagVariation[];
+      const setSegmentInEx = (inEx: FeatureFlagSegmentExclusion | FeatureFlagSegmentInclusion) => {
```

Comment:
what does `inEx` mean? inclusion / exclusion?

Reply:
Yeah, I'd meant to eventually give that a better name. Good catch